### PR TITLE
[Author] Add froala-design-blocks w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by stefanneculai**
Pull request for issue: #
Related issue(s): froala/design-blocks#27 froala/design-blocks#8

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

https://www.producthunt.com/posts/froala-design-blocks-2

# Profile of the lib
 * Git repository (required): https://github.com/froala/design-blocks
 * Official website (optional, not the repository): https://www.froala.com/design-blocks
 * NPM package url (optional): https://www.npmjs.com/package/froala-design-blocks
 * License and its reference: https://github.com/froala/design-blocks/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 197
   - Count of stars: 7012
   - Count of forks: 469
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by stefanneculai** [comment]
It appears there is a problem with the PR, however, I cannot figure it out. Can anyone help a little bit? Thanks.

**Comment by PeterDaveHello** [comment]
@stefanneculai you need to add "filename" info in package.json to point to the main file.

File: ajax/libs/froala-design-blocks/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "website design",
+    "html5",
+    "trending design"
+  ],
+  "license": "https://github.com/froala/design-blocks/blob/master/LICENSE",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/froala/design-blocks.git"
+  },
+  "filename": "dist/css/froala_blocks.css",
```
Comment (PeterDaveHello) [inline ajax/libs/froala-design-blocks/package.json]: The `dist/` should be removed as its the bastPath in the auto-update config.

File: ajax/libs/froala-design-blocks/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "froala design blocks",
+    "html design",
+    "bootstrap",
+    "material design",
+    "html templates",
+    "website design",
+    "html5",
+    "trending design"
+  ],
+  "license": "https://github.com/froala/design-blocks/blob/master/LICENSE",
```
Comment (maruilian11) [inline ajax/libs/froala-design-blocks/package.json]: @extend1994 @sufuf3 I forget how we dill with custom license. Do we use a object for it? Thanks.

Comment (stefanneculai) [reply ajax/libs/froala-design-blocks/package.json]: Looking forward to updates so that I can make the necessary changes. Thanks in advance.

Comment (stefanneculai) [reply ajax/libs/froala-design-blocks/package.json]: @maruilian11 @extend1994 @sufuf3 any suggestions on this? Thanks.

Comment (maruilian11) [reply ajax/libs/froala-design-blocks/package.json]: @PeterDaveHello According to [npm package.json doc](https://docs.npmjs.com/files/package.json), using objects for licenses has been deprecated. Now the way for a license that hasn't been assigned an SPDX identifier is:
``` 
"license" : "SEE LICENSE IN <filename>"
```
But that means we may need to add custom license file to cdnjs, too.
Can you give me some advice? Thanks.

Comment (PeterDaveHello) [reply ajax/libs/froala-design-blocks/package.json]: Just use:
```js
"license": {
  "type": "Custom license",
  "url": "https://github.com/froala/design-blocks/blob/master/LICENSE"
}
```

we are not npmjs ...

Comment (PeterDaveHello) [reply ajax/libs/froala-design-blocks/package.json]: @stefanneculai so is it kind of open source license?

Comment (stefanneculai) [reply ajax/libs/froala-design-blocks/package.json]: That is correct. I'll update the license shortly.

Comment (PeterDaveHello) [reply ajax/libs/froala-design-blocks/package.json]: Thanks!

**Comment by stefanneculai** [comment]
@PeterDaveHello @maruilian11 I just did all the requested changes. Can this be merged now? Thanks in advance.

**Comment by PeterDaveHello** [comment]
@stefanneculai we'll take care of it soon, thanks!

**Comment by PeterDaveHello** [comment]
@stefanneculai looks like minified css should be provided, but it actually not, anything wrong?

Ref: https://github.com/froala/design-blocks/tree/1.0.1#whats-included

**Comment by stefanneculai** [comment]
@PeterDaveHello yes, the minified file is missing. We're going to add it shortly and I will make a PR to change the package.json.

**Comment by PeterDaveHello** [comment]
Cool, let me know if I can help anything 👍

**Pull request merged**